### DISCUSSION TITLE
Response Status Code does not indicate success: 403 (rate limit exceeded)

### DISCUSSION THREAD

**Discussion by hyp36rmax** [RetroDriven/Pocket_Updater#50] [Q&A]
Receiving an error when downloading cores. Is this expected behavior?

"Response Status Code does not indicate success: 403 (rate limit exceeded)"

**Comment by mattpannella** [comment]
try again after 30min or put a github personal access token into the settings

**Comment by hyp36rmax** [comment]
Confirmed working after another attempt allowing substantial time to pass. Thank you!
